### DISCUSSION TITLE
Lack of data versioning

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#39] [General]
The INSPIRE attributes beginLifespanVersion and endLifespanVersion allow to discriminate changes between different versions of the same dataset. The lack of this information (see “POPIMPACT_Non-mandatory attributes”), and more namely beginLifespanVersion, makes very difficult the automatic identification of the changes that occurred over time. Forcing a comparison between all the elements of each of the data versions to find the discrepancies.

**Comment by thorsten-reitz** [comment]
Agreed, most data sets are not properly versioned/historised. There is also the version part of the INSPIRE Identifier that is not used commonly.

**Comment by AndersOstmanNovogit** [comment]
This issue is not yet reported in D4.3.

The issue described is a more specialized version of #38. So we might include it there.

Then I did not fully understand what was the problem. Were you intending to detect differences between two data sets, a new one vs an older one / detect changes? Or why did you want to analyse beginLifespanVersion? How did you solve the problem? Comparing one-by-one?

**Comment by MayteTDGeograma** [comment]
Yes, we were trying to detect differences between two building datasets in order to assign population only to the areas that had changed.
This was solved by repeating the analysis over the whole area, but a better use of versioning could reduce processing times.

**Comment by AndersOstmanNovogit** [comment]
This problem is now reported in section 4.2.3 in D3.4

**Reply by AndersOstmanNovogit** [reply]
Now moved to D4.3 section 4.3.2
